isis/tests/ColorTests.cpp

```diff
@@ -0,0 +1,81 @@
+#include "gtest/gtest.h"
+
+#include "Color.h"
+#include "IException.h"
+
+#include <QDebug>
+#include <QColor>
+
+
+namespace Isis{
+  
+  
+  class QStringQColorPair: public ::testing::TestWithParam<std::pair<QString, QColor> > {
+    // Intentionally left empty
+  };
+  
+  
+  class InvalidColorString: public ::testing::TestWithParam<QString> {
+  // Intentionally left empty
+  };
+
+
+  TEST_P(QStringQColorPair, ValidColorToString) {
+    EXPECT_EQ(GetParam().first, Color::toRGBAString(GetParam().second));
+  }
+  
+  
+  TEST (ColorTests, InvalidColorToString) {
```

Comment:
Do we want spaces between TEST and () to follow ISIS standards?

Reply:
@kdl222 This is a good question. If we do decide to do this, we should update the in-progress [Getting Started with testing](https://github.com/USGS-Astrogeology/ISIS3/wiki/Getting-started-with-testing-(gtest)) guide.

Reply:
Doesn't the ISIS style guide say no space? https://isis.astrogeology.usgs.gov/documents/CodingStandards/CodingStandards.html#methodDefinitions